### ISSUE TITLE
When logged in presenting no cache

### ISSUE THREAD

**Issue by lfbn** [BenjaminAdams/wp-redis-cache#46]
Hi there,

I think it makes more sense to logged in users not presenting cached versions. What do you think?

For example, presenting the cached version will not show you the admin toolbar at the start of the page.

If you agree can I submit a pull request?

Thanks!

**Comment by BenjaminAdams** [comment]
Yeah, that sounds like a nice feature,  I will be awaiting the PR
